homeassistant/util/dt.py

```diff
@@ -41,6 +41,18 @@ def utcnow():
     return dt.datetime.now(UTC)
 
 
+def beatsnow():
```

Comment:
What do you mean common enough? Basically, it's an extension of datetime module to report time in .beats. Why should a class level function reside in a component code? What if I want to reuse .beats for another component, or a script? Should I copy and paste that function all over the place then?